unlock-app/src/components/creator/lock/KeyList.js

```diff
@@ -3,26 +3,37 @@ import { connect } from 'react-redux'
 import styled from 'styled-components'
 import UnlockPropTypes from '../../../propTypes'
 import { expirationAsDate } from '../../../utils/durations'
+import Pagination from '../../interface/pagination/Pagination'
 
 // TODO add pagination
 export function KeyList({ keys }) {
+  const renderItems = items => {
+    return items.map(item => {
+      return (
+        <Row key={item.id}>
+          <Data>{item.transaction}</Data>
+          <Cell>{expirationAsDate(item.expiration)}</Cell>
+          <Data>{item.key}</Data>
```

Comment:
that would be `item.data`. Also do you mind actually renaming the `item` variable into `key` as it is more explicit (or `lockKey` since `key` is also used by jsx)